mtp_api/apps/core/tests/test_dump_for_ap.py

```diff
@@ -48,20 +50,24 @@ def test_valid_arguments(self, mocked_get_modified_records):
         self.assertEqual(before.date(), datetime.date(2019, 9, 1))
 
     def test_empty_results(self):
-        with captured_stdout() as stdout:
-            call_command('dump_for_ap', 'credits', '-')
-        stdout = stdout.getvalue().strip()
-        lines = stdout.splitlines()
-        self.assertEqual(len(lines), 1)
+        filename = tempfile.NamedTemporaryFile().name
```

Comment:
Maybe use `tempfile.NamedDirectory`?

Reply:
possibly. or at least it should be consistent with the tests below such that the temporary file remains in existence rather than relying on GC kicking in late and thereby actually deleting the file produces by the next line

Reply:
Yeah this makes sense

Reply:
Refactored this to be consistent with the other uses, it does make more sense now.